merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/ActivityRecord.java

```diff
@@ -8,13 +8,13 @@
 import java.util.Map;
 import java.util.Optional;
 
-/*package-local*/ record SimulatedActivityRecord(
+/*package-local*/ record ActivityRecord(
```

Comment:
My reasoning here was that an unfinished activity is not necessarily a simulated activity (`SimulatedActivityRecord`). The record just needs to have a fairly 1-1 correspondence with what's inside the DB, which can be a simulated or an unfinished activity depending on the presence of a `duration`.

Let me know how you feel about that. I'm open to keeping this `SimulatedActivityRecord` if that's preferred :)

Reply:
Hmm, I'm having trouble wrapping my head around `an unfinished activity is not necessarily a simulated activity`, maybe we have slightly different definitions of those two terms in mind. I think I'm thinking of "unfinished" as "partially simulated". How does that agree/defer from your mental model?

Reply:
Yeah, to me "partially simulated" is not "simulated", in the same sense that something "partially complete" would not be called "complete".

Reply:
I see - thanks for helping me understand - now our mental models are aligned, if not our nomenclature :+1:

I'm happy to agree to disagree on this one - I'll just write down my thought process

The concepts that I'm wrapping my brain around are:
1. Activity types
2. Activity instances in the plan
3. The result of simulating an activity instance that has not completed
4. The result of simulating an activity instance that has completed

The postgres action representations of these concepts in this PR are:
1. `String` (also stored in the `activity_type` table w/ValueSchema)
2. `ActivityInstanceRecord`
3. `ActivityRecord` where `duration` and `computedAttributes` are both `Optional.empty()`
4. `ActivityRecord` where `duration` and `computedAttributes` are both `Optional.of(...)`

So - I guess I feel like 3 and 4 should somehow signify that they're the result of running a simulation, as opposed to 1 and 2 🤔 Out of context I would think that `ActivityRecord` refers to `ActivityInstanceRecord`...

Perhaps `SimulatedOrPartiallySimulatedActivityRecord` 😅

Reply:
My 2 cents; I think we do need to indicating that the record's usage context is downstream from a simulation (whether completed or uncompleted). I'm happy to find a new name, but in the instance that we can't, I think staying with `SimulatedActivityRecord` should be our fallback position.

Reply:
>So - I guess I feel like 3 and 4 should somehow signify that they're the result of running a simulation, as opposed to 1 and 2

Ahh I see your angle here. I avoided this since I wanted to keep as much domain logic out of our "get activities action" as possible (such as determining whether an activity w/ a `null` duration is unfinished or not). If you're comfortable with that logic residing within the PG action then I can make that change :)

No super strong opinion from me, just trying to stick with convention, which seems to avoid embedding domain logic and exposing `*Record` DB data with a 1-1 mapping with the associated DB table. This is why I opted for embedding this duration `null` check logic in the PG results cell repo.

>My 2 cents; I think we do need to indicating that the record's usage context is downstream from a simulation (whether completed or uncompleted).

In this PR the simulated and unfinished activities are stored respectively in `SimulationResults.simulatedActivities` and `SimulationResults.unfinishedActivities`, so this information is available downstream from a simulation.

Reply:
@mattdailis Also note that 2 and 3 above are for different use cases (one is grabbing activity instances from a plan, the other is grabbing activities recorded within a sim.'s `span`).

Reply:
> @mattdailis Also note that 2 and 3 above are for different use cases (one is grabbing activity instances from a plan, the other is grabbing activities recorded within a sim.'s span).

☝️ yes, exactly - I think my point is I'd like to tag the record classes with the use case they correspond to. Here's where my confusion comes from - when I open up the `postgres` package, I see `ActivityInstanceRecord` and `ActivityRecord`, and I pause - what's the difference between an Activity Instance and an Activity?

![Screen Shot 2022-05-11 at 3 33 44 PM](https://user-images.githubusercontent.com/1189602/167958401-72fbc281-8954-4cd1-8c85-a62babf673fe.png)

Reply:
Conceivably, we could have the Records match the table names, opting for `SpanRecord`

Reply:
Big thanks to @mattdailis for taking the time to help nail this down. We've settled on returning `SpanRecord`s from the DB. If/when "span" gets renamed this can also be renamed, but that's a separate topic.

Latest push attempts to make a better a distinction between planned activities (in `ActivityInstanceRecord`) and spans (simulated/unfinished activities, in `SpanRecord`).